locksmith/migrations/20231025111934-add_checkout_config_to_events.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+
+const table = 'EventData'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn(table, 'checkoutConfigId', {
```

Comment:
Adding `checkoutConfigId` which will be used to point to locks but also store a specific config that can easily be updated!